cypress/e2e/po/pages/cluster-manager/node-templates.po.ts

```diff
@@ -1,6 +1,6 @@
 import PagePo from '@/cypress/e2e/po/pages/page.po';
 import EmberSortableTablePo from '@/cypress/e2e/po/components/ember/ember-sortable-table.po';
-import EmberModalAddNodeTemplatePo from '@/cypress/e2e/po/components/ember/ember-modal-add-node-template.po';
+import EmberModalAddNodeTemplatePo from '@/cypress/e2e/po/components/ember/ember-modal-add-node-template-aws.po';
```

Comment:
There probably needs to be a split here.

This is a generic file handling all providers. The add-node-template one is now provider specific

Reply:
ah good call out! will update that